cppcheck unable to run

Attempted to use [cppcheck](https://github.com/danmar/cppcheck), however, the below stops it.

https://github.com/LoneGazebo/Community-Patch-DLL/blob/fad6e56e04b8fdc8d3efa9e0cba3ea250b32db5c/CvGameCoreDLL_Expansion2/CvUnitClasses.h#L254

> Community-Patch-DLL\CvGameCoreDLL_Expansion2\CvUnitClasses.h:254:0: error: No pair for character ('). Can't process file. File is either invalid or unicode, which is currently not supported. [preprocessorErrorDirective]
#endif'
^

git blame: https://github.com/LoneGazebo/Community-Patch-DLL/blame/fad6e56e04b8fdc8d3efa9e0cba3ea250b32db5c/CvGameCoreDLL_Expansion2/CvUnitClasses.h#L254

Removing it, allows cppcheck to continue.

Note: C++03

--- a/CvGameCoreDLL_Expansion2/CvUnitClasses.h
+++ b/CvGameCoreDLL_Expansion2/CvUnitClasses.h
@@ -251,7 +251,7 @@ public:
 	int* GetUnitNewEraCombatTypeChangesArray(int i);
 	int GetUnitNewEraPromotions(int i, int j) const;
 	int* GetUnitNewEraPromotionsChangesArray(int i);
-#endif'
+#endif
 #if defined(MOD_UNITS_RESOURCE_QUANTITY_TOTALS)
 	int GetResourceQuantityTotal(int i) const;
 #endif

